api/src/controllers/animeController.ts

```diff
@@ -0,0 +1,82 @@
+import type { Request, Response, NextFunction } from '@@types/express.js';
+import { errors, entities } from '@@utils/index.js';
+import { Anime } from '@@entities/index.js';
+
+interface Params {
+    animeId: number
+};
+
+// Create
+async function create(req: Request, res: Response<Params>, next: NextFunction) {
+    const { params } = res.locals;
+    const { animeId } = params as Params;
+
+    const [anime, animeErr] = await entities.insert<Anime>(Anime, req.body);
+
+    if(animeErr) {
```

Comment:
You should also check if `anime` exists here. There's a method in the `errors` utils for this

```ts
if(animeErr || !anime) {
    return errors.sendEntitiesReponse({
        res,
        err: animeErr,
        message: "Error finding Anime",
        entityReturn: anime,
        missingEntityReturnMessage: "Unable to find Anime"
    });
}
```